docs/source/developer_guides/low_level_api.md

```diff
@@ -84,8 +84,8 @@ DummyModel(
   (lm_head): Linear(in_features=10, out_features=10, bias=True)
 )
 ```
-Note that it should be up to users to properly take care of saving the adapters (in case they want to save adapters only), as `model.state_dict()` will return the full state dict of the model.
-In case you want to extract the adapters state dict you can use the `get_peft_model_state_dict` method:
+
+To only save the adapter, use the [`get_peft_model_state_dict`] function
```

Comment:
missing colon (:) at the end of line 88 (?)

Reply:
Good catch!